manifests/server/export.pp

```diff
@@ -57,7 +57,7 @@
 # @param v4_export_name
 # @param nfsv4_bindmount_enable
 #
-# @examples
+# @example
```

Comment:
Ideally every example has a description, otherwise puppet-strings doesn't render it well.